src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -48,6 +49,19 @@
     private final BigQueryPdao bigQueryPdao;
     private final RestHighLevelClient client;
 
+    private final Map<String, String> columnReplacements = Map.of(
```

Comment:
The only problem with this is that we have to modify code every time we add a new column type to the query. Now that `encode()` won't modify any field that doesn't need encoding is this table still needed?

Reply:
Could this be moves to configuration somehow?  Maybe now isn't the right time for that

Reply:
I think the table can be removed if we call `encode()` on all data that's matched using the `AS` regex. Any data that doesn't require encoding won't be affected.